darwin/future/tests/meta/queries/test_item.py

```diff
@@ -70,3 +70,38 @@ def test_delete(
             json={},
         )
         item_query.delete()
+
+
+def test_archive(
+    item_query: ItemQuery, items_json: List[dict], items: List[Item]
+) -> None:
+    with responses.RequestsMock() as rsps:
+        rsps.add(
+            rsps.GET,
+            item_query.client.config.api_endpoint + "v2/teams/test/items",
+            match=[
+                query_param_matcher(
+                    {"page[offset]": "0", "page[size]": "500", "dataset_ids": "1"}
+                )
+            ],
+            json={"items": items_json, "errors": []},
+        )
+        team_slug = items[0].meta_params["team_slug"]
+        dataset_id = items[0].meta_params["dataset_id"]
+        rsps.add(
+            rsps.POST,
+            items[0].client.config.api_endpoint + f"v2/teams/{team_slug}/items/archive",
+            status=200,
+            match=[
+                json_params_matcher(
+                    {
+                        "filters": {
+                            "item_ids": [str(item.id) for item in items],
+                            "dataset_ids": [dataset_id],
+                        }
+                    }
+                )
+            ],
+            json={},
+        )
+        item_query.archive()
```

Comment:
Sad path opp here too.

Reply:
@owencjones Similar to my comment on the restore meta ticket and after discussion w/ Nathan, there is only one path this function can take due to no argument & no return

Will proceed with this test alone as error responses are tested by the underlying Core method, but let me know if you disagree